src/init.cpp

```diff
@@ -1488,6 +1488,13 @@ bool AppInit2(boost::thread_group& threadGroup, CScheduler& scheduler)
         uiInterface.NotifyBlockTip.disconnect(BlockNotifyGenesisWait);
     }
 
+#ifdef ENABLE_WALLET
```

Comment:
It's unfortunate that this has to introduce a new `#ifdef ENABLE_WALLET` here, and ref to pWalletMain.
We're trying to get rid of them to remove the circular dependency between` _wallet` and `_server` libs (#7965), and in the interest of multi-wallet support.